packages/core/template/project/Design.md

```diff
@@ -0,0 +1,100 @@
+---
+  layout: default.md
+  title: "Design"
+  pageNav: 3
+---
+
+<!-- * Table of Contents -->
+<page-nav-print />
+
+<!-- -------------------------------------------------------------------------------------------------------------------- -->
+
+# Design
+
+<box type="tip">
+This section describes the architecture of your app, explaining how the main components work and interact with each other. Using architecture diagrams is recommended.
+</box>
+
+### Architecture
+
+The ***Architecture Diagram*** given below explains the high-level design of the ProjectEx App.
+
+{{ johnDoeVariable }}
+<box type="info" seamless>
+
+Replace the above dummy `img` component with an Architecture Diagram of your project created using [`puml`](https://markbind.org/userGuide/components/imagesAndDiagrams.html#diagrams) feature.
```

Comment:
One example PUML file would be cool (I remember as a user this tripped me up before so sample code wld be great)